student.rb

```diff
@@ -0,0 +1,10 @@
+class Student < Person
+  def initialize(name, age, parent_permission, classroom)
+    super(name, age, parent_permission)
+    @classroom = classroom
+  end
+
```

Comment:
- [ ] Student class is well created , however , it should have the optional (or default) value for `name` and for `parent_permission` parameters as describe to in the parent class, Kindly add them 👍 

```ruby
class  Student < Person
 .
 .
 .
  def initialize(age, classroom, name = "Unknown", parent_permission = true)
       #...
  end
 .
 .
 .
end
```